packages/utils/telemetry-utils/src/logger.ts

```diff
@@ -48,6 +48,11 @@ export interface ITelemetryLoggerPropertyBags{
     error?: ITelemetryLoggerPropertyBag;
 }
 
+export type TelemetryEventTypes =
```

Comment:
I'm not sure you need this, I think you want to update all signatures in this file that use `ITelemetryBaseEvent` (or this `TelemetryEventTypes`) to use `ITelemetryEvent` instead (mentioned in a comment in the definitions file)